source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -1097,27 +1118,171 @@ List of registry entries to be monitored. One entry per line. Multiple lines may
++                          +                                                                                                                       +
+|                          | A list of denied and allowed permissions will be given for each user or group.                                        |
++                          +                                                                                                                       +
+|                          | .. versionadded:: 4.1.0                                                                                               |
++                          +------------------------------------------------------------+----------------------------------------------------------+
+|                          | Default value                                              | yes                                                      |
++                          +------------------------------------------------------------+----------------------------------------------------------+
+|                          | Allowed values                                             | yes, no                                                  |
++--------------------------+------------------------------------------------------------+----------------------------------------------------------+
+| **check_mtime**          | Check the modification time of a registry.                                                                            |
++                          +                                                                                                                       +
+|                          | .. versionadded:: 4.1.0                                                                                               |
++                          +------------------------------------------------------------+----------------------------------------------------------+
+|                          | Default value                                              | yes                                                      |
++                          +------------------------------------------------------------+----------------------------------------------------------+
+|                          | Allowed values                                             | yes, no                                                  |
++--------------------------+------------------------------------------------------------+----------------------------------------------------------+
+| **check_type**           | Check the type of a value. It is used to notify changes in the values of the monitored registry.                      |
++                          +                                                                                                                       +
+|                          | This is limited to REG_SZ, REG_MULTI_SZ, REG_DWORD, REG_DWORD_BIG_ENDIAN, REG_QWORD.                                  |
```

Comment:
The full list of supported types is larger, it can be found [here](https://github.com/wazuh/wazuh/blob/2792f62428af613aabf4cdcca802e04fe91e05f3/src/syscheckd/registry/events.c#L19-L32)